test/common/include/alpaka/test/mem/view/Iterator.hpp

```diff
@@ -35,10 +37,10 @@ namespace alpaka
                     template<
                         typename T,
                         typename TSource>
-                    using MimicConst = typename std::conditional<
+                    using MimicConst = std::conditional_t<
                         std::is_const<TSource>::value,
-                        typename std::add_const<T>::type,
-                        typename std::remove_const<T>::type>;
+                        std::add_const_t<T>,
+                        std::remove_const_t<T>>;
```

Comment:
This is an alias template. You probably have to adjust the call to `typename MimicConst<T, TSource>::type` as well now to make the compiler happy.